_pages/for-devs.md

```diff
@@ -192,7 +192,7 @@ permalink: /for-devs
 </div>
 <div class="row no-gutters position-relative">
     <div class="col-md-4 mb-md-0 p-md-4">
-        <img src="/assets/images/library/thumb-designassets.png" class="w-100" alt="...">
+        <img src="/assets/images/library/thumb-designassets.png" class="w-100" alt="ChRIS design assets graphic.">
```

Comment:
Nitpick: would you remove the trailing period? None of the other alt tags are complete sentences so let's try to be consistent.

Reply:
Thanks, sure!